15-swiftui-with-combine/projects/final/HNReader/View/SettingsView.swift

```diff
@@ -83,11 +83,11 @@ struct SettingsView: View {
   
   private func moveKeyword(from source: IndexSet, to destination: Int) {
     guard let source = source.first,
-          destination != settings.keywords.endIndex else { return }
+          destination != source else { return }
 
-    settings.keywords
-      .swapAt(source,
```

Comment:
`.swap` won't work here because when reordering the top keyword to the bottom, we don't want the bottom one to go to the top, it can just stay at the bottom below the reordered keyword